lectures/intro_supply_demand.md

```diff
@@ -53,6 +53,12 @@ kernelspec:
 import numpy as np
 import matplotlib.pyplot as plt
 from collections import namedtuple
+from matplotlib import font_manager
+import matplotlib.font_manager
+
+# List all available font names
+font_list = [f.name for f in matplotlib.font_manager.fontManager.ttflist]
+print(font_list)
```

Comment:
@longye-tian do we need these lines and information?

Reply:
Hi Matt @mmcky ,

This is just to know what font are available. So they are temporary, just for me to fix things later. So they will be deleted at the end.

I'm still trying to workout a solution for this problem.

I'll let the team know if I fix it.

Best,
Longye

Reply:
ah sorry - thanks @longye-tian I thought this was ready. My mistake. Take your time. Thanks for your work.